library/x509_crt.c

```diff
@@ -43,13 +43,13 @@
 #include "mbedtls/threading.h"
 #endif
 
-#if defined(MBEDTLS_HAVE_TIME)
+#if defined(MBEDTLS_HAVE_TIME) || defined(MBEDTLS_FS_IO)
```

Comment:
What about to rather add `#include windows.h` after line 67 in the `#if defined(MBEDTLS_FS_IO)` section just below?

Otherwise, to accept your commit we need a DCO in the commit message.